src/TimeESPeasy.ino

```diff
@@ -3,19 +3,12 @@
    Time stuff
  \*********************************************************************************************/
 
-#define SECS_PER_MIN  (60UL)
-#define SECS_PER_HOUR (3600UL)
-#define SECS_PER_DAY  (SECS_PER_HOUR * 24UL)
-#define DAYS_PER_WEEK (7UL)
-#define SECS_PER_WEEK (SECS_PER_DAY * DAYS_PER_WEEK)
-#define SECS_PER_YEAR (SECS_PER_WEEK * 52UL)
-#define SECS_YR_2000  (946684800UL) // the time at the start of y2k
-#define LEAP_YEAR(Y) (((1970 + Y) > 0) && !((1970 + Y) % 4) && (((1970 + Y) % 100) || !((1970 + Y) % 400)))
 #include <time.h>
+#include "TimeESPeasy.h"
 
 struct tm tm;
 uint32_t  syncInterval = 3600;       // time sync will be attempted after this many seconds
-double    sysTime = 0.0;             // Use high resolution double to get better sync between nodes when using NTP
+// double    sysTime = 0.0;             // Use high resolution double to get better sync between nodes when using NTP
```

Comment:
Why do you remove `sysTime`?

Reply:
> 
> 
> Why do you remove `sysTime`?

I moved it to TimeESPeasy.cpp to make it available from Time.cpp.

Reply:
Reverted back.

Reply:
OK, maybe we should then get rid of TimeESPeasy.ino and move the global defined variables to a separate .h and .cpp file in the Globals directory.

Then global defined variables must in the .h file have the `extend` keyword and only declared once in their .cpp file.
See other .h/.cpp examples in the Globals directory.

The tm struct should then be defined in DataStructs directory and its global variable in the Globals, as it is a global defined variable to keep track of the current time.

The reason I'm transitioning .ino files into their .h/,cpp is mainly because of build issues we're seeing with .ino files.
Also it does make it clear where things are defined, when they are included, etc. So then we can use the tools like cpp-check and the compiler to check our code, which is much harder on .ino files.

Reply:
That sounds reasonable, but isn't that better to make refactoring separately from other stuff?

Reply:
Well you are moving things around... So we're already mixing it.
Sometimes adding features and refactoring are not that black-and-white (well it is more rare it is clearly a different process)